apps/docs/content/getting-started/releases-versioning.mdx

```diff
@@ -15,51 +15,75 @@ Unlike many JavaScript packages distributed on [NPM](https://www.npmjs.com/), th
 
 {/* START AUTO-GENERATED CHANGELOG */}
 
-## Current release: [v3.7.0](/releases/v3.7.0)
+## Current release: [v3.8.0](/releases/v3.8.0)
```

Comment:
Are the changes to this file intentional?

Reply:
I think we forgot to push a change when dropping 3.8